demo/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,8 @@
         android:icon="@mipmap/ic_launcher"
         android:label="Browser Switch Demo">
 
-        <activity android:name=".DemoActivity">
+        <activity android:name=".DemoActivity"
+            android:launchMode="singleTask">
```

Comment:
We can't change this because it is demonstrating how a current integration is working in the wild. If we see odd behavior with DemoActivity it shows we're making a breaking change. Also changing this shows that we're adding a breaking change to the library. Ideally this activity would not change until a major version bump.

We'll need a way for consumers to opt into this new behavior flow.

Reply:
That’s ok, I understand and completely agree. Feel free to use this PR for inspiration to implement a more complete solution with a feature toggle.

I'm sorry that I can't implement the full solution with the feature toggle myself. My employer says I've already spent too much time on this already 😃 and our fork is working well for us for now.